tests/openbb_terminal/econometrics/txt/test_econometrics_controller/test_call_bgod[other1].txt

```diff
@@ -0,0 +1,4 @@
+
+The following args couldn't be interpreted: ['-n', 'dataset']
```

Comment:
Is there an issue there ?

Reply:
@Chavithra can you try with my changes and let me know if that resolves it.. haha get it? I'll walk myself out...

Reply:
@Chavithra this is not an issue.